doc/releases/changelog-dev.md

```diff
@@ -49,6 +49,10 @@
 
 <h3>Improvements 🛠</h3>
 
+* `Projector` now accepts a state vector representation, which enables the creation of projectors
+  in any basis.
+  [(#4192)](https://github.com/PennyLaneAI/pennylane/pull/4192)  
+
```

Comment:
@BorjaRequena please could you add a small code example?

Reply:
Can be as simple as copying the code block in the example further down, which already looks great.